src/scriptworker/constants.py

```diff
@@ -232,8 +232,7 @@
                                     [
                                         r"^(?P<path>/mozilla-mobile/"
                                         "(?:"
-                                        "fenix"
-                                        "|fennec-profile-manager"
+                                        "fennec-profile-manager"
```

Comment:
Not a blocker: I filed [bug 1819863](https://bugzilla.mozilla.org/show_bug.cgi?id=1819863) to remove this access.